chef/cookbooks/crowbar/recipes/prepare-upgrade-scripts.rb

```diff
@@ -297,7 +298,24 @@
   owner "root"
   group "root"
   variables(
-    nova_controller: roles.include?("nova-controller")
+    nova_controller: roles.include?("nova-controller"),
+    remotes_present: !remote_nodes.empty?,
+    remote_node: remote_node,
```

Comment:
Just a nit ... for the code clarity it might be better to rename `remote_node` to `is_remote_node`...

Reply:
Done.